src/test/java/reactor/netty/http/HttpOperationsTest.java

```diff
@@ -90,4 +94,28 @@ public void httpAndJsonDecoders() {
 		assertThat(t, nullValue());
 	}
 
+	@Test
+	public void testIssue948() {
+		assertEquals("", HttpOperations.resolvePath("http://localhost:8080"));
+		assertEquals("", HttpOperations.resolvePath("http://localhost:8080/"));
+		assertEquals("/", HttpOperations.resolvePath("http://localhost:8080//"));
+		assertEquals("a", HttpOperations.resolvePath("http://localhost:8080/a"));
+		assertEquals("a", HttpOperations.resolvePath("http://localhost:8080/a/"));
+		assertEquals("", HttpOperations.resolvePath("http://localhost:8080/?b"));
+		assertEquals("a", HttpOperations.resolvePath("http://localhost:8080/a?b"));
+		assertEquals("", HttpOperations.resolvePath("http://localhost:8080/#b"));
+		assertEquals("a", HttpOperations.resolvePath("http://localhost:8080/a#b"));
+		assertEquals("a", HttpOperations.resolvePath("http://localhost:8080/a?b#c"));
+
+		assertEquals("", HttpOperations.resolvePath(""));
+		assertEquals("", HttpOperations.resolvePath("/"));
+		assertEquals("/", HttpOperations.resolvePath("//"));
```

Comment:
I realize this is preexisting behavior but the Javadoc says the leading and trailing slash are removed if present, but here only the leading slash is removed. Is the second slash not considered a trailing slash? It also means the following two have the same result :

```java
assertEquals("/", HttpOperations.resolvePath("//"));
assertEquals("/", HttpOperations.resolvePath("///"));
```